### DISCUSSION TITLE
AltLinux Ltsp

### DISCUSSION THREAD

**Discussion by svetoi1985** [ltsp/ltsp#819] [Q&A]
Hello, I need help installing and configuring Ltsp on alt linux. The site has a shortened version of the installation without a detailed description.

**Comment by alkisg** [comment]
Hi, currently LTSP supports Debian-based distributions:
https://ltsp.org/docs/installation/

AltLinux appears to use rpm and [make-initrd](https://en.altlinux.org/Make-initrd), so it's not currently supported. A developer will need to add support for it before it can be used by end users.
